utils/kata-manager.sh

```diff
@@ -782,6 +821,25 @@ handle_docker()
 	docker --version
 }
 
+handle_nerdctl()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous $containerd_project configuration"
+		[ -e "$containerd_config" ] && sudo mv $containerd_config $containerd_config.system-$(date -Iseconds)
```

Comment:
It's a bit more work, but `configure_containerd()` also backs up config files, so we could create a `backup_containerd_backup()`.